src/parachain/nomination.ts

```diff
@@ -188,22 +190,36 @@ export class DefaultNominationAPI extends DefaultTransactionAPI implements Nomin
         return nonceMap;
     }
 
-    async listNominatorsRaw<C extends CurrencyUnit>(
-        currency: Currency<C>
+    async listAllNominations(): Promise<
+        [[WrappedCurrency, [number, string, string]], MonetaryAmount<Currency<CollateralUnit>, CollateralUnit>][]
+        > {
+        let result: [
+            [WrappedCurrency, [number, string, string]],
+            MonetaryAmount<Currency<CollateralUnit>, CollateralUnit>
+        ][] = [];
+        for (const currency of WRAPPED_CURRENCIES) {
```

Comment:
Also, slight implementation note, instead of using a for loop here, better to use a `map` function that can execute the await call in line 201 in parallel otherwise you will execute it sequentially.